doc/move_group_interface/src/move_group_interface_tutorial.cpp

```diff
@@ -360,38 +360,75 @@ int main(int argc, char** argv)
   visual_tools.trigger();
   visual_tools.prompt("next step");
 
-  // Now, let's attach the collision object to the robot.
+  // Attaching Objects
+  // ^^^^^^^^^^^^^^^^^
+  //
+  // Now, let's attach a different type of collision object to the robot, for the purposes of "manipulating" it.
+  moveit_msgs::CollisionObject attachment_object;
```

Comment:
```suggestion
  moveit_msgs::CollisionObject object_to_attach;
```
"attachment_object" reads strangely to me.